pgctl/cli.py

```diff
@@ -410,7 +410,10 @@ def debug(self):
                 'Must debug exactly one service, not: ' + commafy(self.service_names),
             )
 
-        self.stop()
+        if service.state['state'] != 'down':
+            self.stop()
```

Comment:
its idempotent but not a no-op. It emits an unnecessary and confusing "<service> is already stopped" message. It's already down, why bother?